backend/sokdak/src/main/java/com/wooteco/sokdak/notification/domain/Notification.java

```diff
@@ -64,4 +66,10 @@ public String getContent() {
         }
         return getComment().getMessage();
     }
+
+    public void validateOwner(Long accessMemberId) {
+        if (!member.hasId(accessMemberId)) {
+            throw new AuthenticationException();
```

Comment:
```suggestion
            throw new AuthorizationException();
```
인증이 아닌 인가에 대한 예외이니 네이밍 수정하면 좋겠네요!

Reply:
해당 부분은 Authentication이 맞지 않나용...? 토큰은 존재하는데, 다른 사용자의 알림을 삭제하려는 시도입니다!